packages/babel-preset-default/index.js

```diff
@@ -1,22 +1,27 @@
 const env = process.env.BABEL_ENV || process.env.NODE_ENV;
 const isTestEnv = env === 'test';
 
-module.exports = {
-	presets: [
-		! isTestEnv && [ require( 'babel-preset-env' ), {
-			modules: false,
-			targets: {
-				browsers: [ 'extends @wordpress/browserslist-config' ],
-			},
-		} ],
-		isTestEnv && [ require( 'babel-preset-env' ) ],
-	].filter( Boolean ),
-	plugins: [
-		require( 'babel-plugin-transform-object-rest-spread' ),
-		[ require( 'babel-plugin-transform-react-jsx' ), {
-			pragma: 'wp.element.createElement',
-		} ],
-		require( 'babel-plugin-transform-async-generator-functions' ),
-		! isTestEnv && require( 'babel-plugin-transform-runtime' ),
-	].filter( Boolean ),
+module.exports = function() {
+	return {
+		presets: [
+			! isTestEnv && [ '@babel/preset-env', {
+				modules: false,
+				targets: {
+					browsers: [ 'extends @wordpress/browserslist-config' ],
+				},
+				useBuiltIns: 'usage',
+			} ],
+			isTestEnv && [ '@babel/preset-env', {
+				useBuiltIns: 'usage',
+			} ],
+		].filter( Boolean ),
+		plugins: [
+			'@babel/plugin-proposal-object-rest-spread',
```

Comment:
Not sure if the Babel core has been updated for it yet, but Rest/Spread was ratified as part of ES2018, so it might be possible to remove this.

https://github.com/tc39/proposals/blob/master/finished-proposals.md